model/patch/patch.go

```diff
@@ -615,6 +632,10 @@ func (p *Patch) GetRequester() string {
 }
 
 func (p *Patch) CanEnqueueToCommitQueue() bool {
+	if p.HasGitInfo() {
+		return true
+	}
+
 	for _, modulePatch := range p.Patches {
```

Comment:
Do we still need to consider this?

Reply:
I think you're right. I was originally intending to allow users to enqueue patches they've uploaded with the old CLI (that were in mbox format). On second thought it will be confusing if we let them specify a message from spruce and then don't use their message.

I've adjusted it to only enqueue patches with the new (after this PR) CLI, that include the git metadata.